src/js/modules/sections/reducer.js

```diff
@@ -16,132 +10,132 @@ const initialState = {
       slug: "news",
       description: "The news of Stuyvesant.",
       parentSlug: null,
+      permalink: "/news"

```

Comment:
permalinks will be generated when we process response data from API. so, yes, it will be in the selectors in the future.